soe/src/stanford.make

```diff
@@ -212,7 +212,7 @@ projects[stanford_sites_helper][download][url] = "git@github.com:SU-SWS/stanford
 projects[stanford_sites_helper][subdir] = "stanford"
 projects[stanford_sites_helper][type] = "module"
 
-projects[stanford_video][download][tag] = "7.x-2.2+6-dev"
+projects[stanford_video][download][tag] = "7.3.7-alpha.5"
```

Comment:
I don't think there's an upgrade path for this module from 7.x-2.x to 7.x-3.x. Did you intend to change `stanford_news` instead?

Reply:
Whoops. Yes, it looks like I totally added it to the wrong module. Sorry about that. 

I pushed the Fixup!.